tests/test_run_from_cmdline.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+from dirsync import run
+
+
+def test_run_from_cmd_line_then_create_user_config_file_when_not_exist():
+    with patch.object(run, "os") as os_mock:
+        os_mock.path.isfile.return_value = False
+
+        with patch("builtins.open") as mock_file:
+            with patch.object(run, "sync"), patch.object(run, "ArgParser"):
+                run.from_cmdline()
+
+            assert mock_file.called
```

Comment:
wouldn't we want to check a few more things? i.e. with what arguments open() has been called?